test/controllers/git-tab-controller.test.js

```diff
@@ -745,10 +747,9 @@ describe('GitTabController', function() {
         commitMessages = wrapper.find('.github-RecentCommit-message').map(node => node.text());
         assert.deepEqual(commitMessages, ['Initial commit']);
 
+        const expectedCoAuthor = new Author(coAuthorName, coAuthorEmail);
         assert.strictEqual(wrapper.find('CommitView').prop('message'), commitSubject);
-        assert.deepEqual(wrapper.find('CommitView').prop('selectedCoAuthors'), [
-          {name: 'Foo Bar', email: 'foo@bar.com'},
-        ]);
+        assert.deepEqual(wrapper.find('CommitView').prop('selectedCoAuthors'), [expectedCoAuthor]);
```

Comment:
It's really helpful that `deepEqual` works on object instances 😅

Reply:
YUP